pkg/cmd/serve/cmd.go

```diff
@@ -20,6 +41,26 @@ func NewServeCommand(env *environments.Env) *cobra.Command {
 			if err != nil {
 				glog.Fatalf("Unable to initialize environment: %s", err.Error())
 			}
+			// Get the build information:
+			buildInfo, e := getBuildInfo()
+			// Get the commitSha from the api package:
+			version, err := api.GetCommitSha()
+			if err != nil || e != nil {
+				glog.Fatalf("unable to retrieve buildinfo")
```

Comment:
You are no passing parameter to be inserted into the log string, so there is no need to use Fatalf.

```suggestion
				glog.Fatal("unable to retrieve buildinfo")
```

Reply:
Thanks @ziccardi ! Added the changes in the last commit.